SDRNA RNA Mining.ipynb

```diff
@@ -0,0 +1,278 @@
+      "    Key: locus_tag, Value: ['RDN5-6']\n",
+      "    Key: note, Value: ['Variant 5S ribosomal rRNA (5S rRNA) gene; located at a site distal to the RDN1 locus']\n",
+      "    Key: product, Value: ['5S ribosomal RNA']\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    "Purpose: mine the rRNA from Genbank sequences\n",
+    "and output in a fasta file with the accession number, locus tag, \n",
+    "the start, the end, and the direction of the sequence, with\n",
+    "optional database identifiers and optional prodcuts or notes\n",
+    "\n",
+    "The current version looks for the .gb Genbank files in the folder\n",
+    "that is two levels under the main folder\n",
+    "and loops through each Genbank files to look for the rRNA feature.\n",
+    "\"\"\"\n",
+    "\n",
+    "def SDRNA_RNA_Mining(folder,subfolder):\n",
+    "    pathname = Path(folder + \"/\" + subfolder + \"/\")\n",
+    "    filelist = os.listdir(pathname)\n",
+    "    opname = folder + \" \" + subfolder + \" output.fasta\"\n",
+    "    output_file = pathname / opname\n",
+    "    f = open(output_file, \"w\")\n",
+    "    \n",
+    "    for fn in range(len(filelist)):\n",
+    "        if \"gb\" in filelist[fn]: \n",
+    "            gbfile = filelist[fn]\n",
+    "            input_file = pathname / gbfile\n",
+    "\n",
+    "            for rec in SeqIO.parse(input_file, \"gb\"): \n",
+    "                acc = rec.annotations['accessions'][0]\n",
+    "                seq = rec.seq\n",
+    "\n",
+    "                for feature in rec.features:\n",
+    "                    ifrrRNA = 0\n",
+    "\n",
+    "                    if \"rRNA\" in feature.type:\n",
+    "                        print(feature)\n",
+    "\n",
+    "                        seqstart = feature.location.start.position\n",
+    "                        seqend = feature.location.end.position\n",
+    "\n",
+    "                        if feature.location.strand == 1:\n",
+    "                            strdir = \"forward\"\n",
+    "                            rRNAseq = seq[seqstart:seqend]\n",
+    "                        elif feature.location.strand == -1:\n",
+    "                            strdir = \"revcomp\"\n",
+    "                            rRNAseq = seq[seqstart:seqend].reverse_complement()[::]\n",
+    "                        else:\n",
+    "                            strdir = \"\"\n",
+    "\n",
+    "                        if \"db_xref\" in feature.qualifiers:\n",
+    "                            addinfo1 = feature.qualifiers[\"db_xref\"][0]\n",
+    "                        else:\n",
+    "                            addinfo1 = \"\"\n",
+    "                        \n",
+    "                        if \"product\" in feature.qualifiers:\n",
+    "                            addinfo2 = feature.qualifiers[\"product\"][0]\n",
```

Comment:
Same applies for this variable name.

Reply:
Changed to note in [rRNA_mining.ipynb](https://github.com/Zymergen/adv-tech-prog/blob/FZL_merge_on_ATP/Code/rRNA_mining.ipynb) on atp-tech-prog --> FZL_merge_on_ATP branch --> Code